libs/eo/shared/atomic-design/ui-atoms/src/lib/eo-resolution-picker/eo-resolution-picker.component.ts

```diff
@@ -14,6 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/* eslint-disable @nrwl/nx/enforce-module-boundaries */
```

Comment:
Is this required now? It did work beforehand?

Reply:
Workaround for NX making up dumb circular dependency things